src/lib/PageWrapper.ts

```
@@ -14,6 +14,27 @@ export default class PageWrapper {
     this.page = page;
   }
 
+  async goTo(url: string): Promise<void> {
+    try {
+      await this.page.goto(url, {
+        waitUntil: "networkidle0",
+      });
+    } catch (err) {
+      console.log(err);
+    }
+  }
+
+  async enterText(selector: string, text: string): Promise<void> {
+    try {
+      const elementHandle = await this.waitForElementToAppear(selector, {
+        visible: true,
+      });
+      await elementHandle.type(text);
+    } catch (error) {
+      console.log(error);
```

why not `throw new Error`? can you please update

---

> why not `throw new Error`? can you please update

Yes, I will update.
But I didn't throw error here because we are not handling this in `AmazonLoginPage` class. If I use `throw new error` here, shouldn't I use try catch block in `AmazonLoginClass`?

---

stacktrace is provided by the node. We should throw an error if the type function is not working.
We cannot add try and catch on every step function or page function. It doesn't make any sense.

---

please try on local and you get the answer.